.gitignore

```diff
@@ -1,3 +1,4 @@
+composer.lock
```

Comment:
Robo does highest / lowest testing, which requires composer.lock be committed to the repository. Committing the composer.lock also allows the phar build to be reproducible.